server/base/src/main/java/org/apache/accumulo/server/constraints/MetadataConstraints.java

```diff
@@ -379,6 +390,8 @@ public String getViolationDescription(short violationCode) {
         return "Bulk load mutation contains either inconsistent files or multiple fateTX ids";
       case 9:
         return "Invalid data file metadata format";
+      case 10:
+        return "Suspended timestamp can not be negative";
```

Comment:
Could fail for other reasons.

```suggestion
        return "Suspended timestamp is not valid";
```